src/poker.js

```diff
@@ -1,10 +1,111 @@
-function winningPair() {}
+function calculatePairValue(arr) {
+  const cardValues = {
+    2: 2,
+    3: 3,
+    4: 4,
+    5: 5,
+    6: 6,
+    7: 7,
+    8: 8,
+    9: 9,
+    10: 10,
+    J: 11,
+    Q: 12,
+    K: 13,
+    A: 14
+  }
+  return arr.reduce((sum, card) => sum + cardValues[card], 0)
+}
+
+function isPair(arr) {
+  if (arr.length === 2) {
+    if (arr[0] === arr[1]) {
+      return [arr[0], arr[1]]
+    } else {
+      return false
+    }
+  } else if (arr.length === 3) {
+    if (arr[0] === arr[1]) {
+      return [arr[0], arr[1]]
+    } else if (arr[0] === arr[2]) {
+      return [arr[0], arr[2]]
+    } else if (arr[1] === arr[2]) {
+      return [arr[1], arr[2]]
+    } else {
+      return false
+    }
+  }
+}
```

Comment:
This is really difficult to understand at a glance. There are some quick improvements you could make here, such as moving the comparisons to their own variable so they have a descriptive name.

In general if you feel like you have to nest lengthy if / else / if chains then there's likely an easier way to do something